README.md

```diff
@@ -171,14 +171,16 @@ or do not satisfy the requirements, an error will be thrown.
   | [`<RegExpError>`](#regexperror)
   | [`<MissingEnvError>`](#missingenverror)
   | [`<EnumError>`](#enumerror)
+  | [`<RequiredDefaultMutexError>`](#requireddefaultmutexerror)
 
 ### Definition
 
 This is a private prototype that offers the following methods:
 
 #### `Definition#required()`
 
-Specifies that this env var is required. By default, env vars are optional.
+Specifies that this env var is required to have a value (env vars are optional by default). This means
+that a value must be provided, and it **cannot** be the empty string, `''`.
```

Comment:
The PR is labeled as "fix". So I assume the CI/CD tooling will bump `z` in `x.y.z`. The PR should be adjusted so that it is a new major. With that done, I'd argue dropping Node 10 and 12 is viable.

Reply:
The title is old.  I updated the commit to be `refactor` with `BREAKING CHANGE`.

We can drop the old nodes, sure.